src/main/kotlin/uk/gov/justice/digital/hmpps/personrecord/service/message/CreateUpdateService.kt

```diff
@@ -26,17 +36,31 @@ class CreateUpdateService(
   private val eventLoggingService: EventLoggingService,
 ) {
 
+  @Retryable(
+    backoff = Backoff(random = true, delay = 1000, maxDelay = 2000, multiplier = 1.5),
+    retryFor = [
+      ObjectOptimisticLockingFailureException::class,
+      CannotAcquireLockException::class,
+      JpaSystemException::class,
+      JpaObjectRetrievalFailureException::class,
+      DataIntegrityViolationException::class,
+      ConstraintViolationException::class,
+      StaleObjectStateException::class,
+    ],
+  )
```

Comment:
See https://stackoverflow.com/questions/49678581/spring-retry-with-transactional - this seems to indicate that @Retryable will happen around @Transactional